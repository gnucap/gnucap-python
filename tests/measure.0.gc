* Felix Salfelder 2020
* GPLv3+

attach public c_python.so
loadpy measure_at.py

* bug? not used in measure or eval output
options numdgt=5

verilog

resistor #(.r(1)) r1(2,1);
capacitor #(.c(1)) c2(0,1);
vsource #(.dc(1)) v1(2,0);

store tran v(1)
print tran v(1)

parameter xx = .4

dc > /dev/null
tran 1 uic > /dev/null
measure vlin at:0(probe='v(1)' x=xx)
measure vquad at(probe='v(1)' x=xx)

options trtol=1
dc > /dev/null
tran 1 uic > /dev/null
measure vlin at:0(probe='v(1)' x=xx)
measure vquad at(probe='v(1)' x=xx)

options trtol=.1
dc > /dev/null
tran 1 uic > /dev/null
measure vlin at:0(probe='v(1)' x=xx)
measure vquad at(probe='v(1)' x=xx)

options trtol=.01
dc > /dev/null
tran 1 uic > /dev/null
measure vlin at:0(probe='v(1)' x=xx)
measure vquad at(probe='v(1)' x=xx)
