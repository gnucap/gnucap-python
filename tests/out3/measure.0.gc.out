Gnucap : The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2013, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
main version
core-lib version
default plugins
generator: already installed, replacing
stashing as generator:0
transient: already installed, replacing
stashing as transient:0
transient: already installed, replacing
stashing as transient:0
at: already installed, replacing
stashing as at:0
vlin= 0.326063084479988
vquad=0.32951611226786853
vlin= 0.32848423483441
vquad=0.330018641313625
vlin= 0.329402443156465
vquad=0.329745189381181
vlin= 0.329629474703186
vquad=0.3296908487626662

